pkg/kv/kvpb/api.proto

```diff
@@ -3014,6 +3014,7 @@ message Header {
 
 message WriteOptions {
   uint32 origin_id = 1[(gogoproto.customname) = "OriginID"];
+  util.hlc.Timestamp origin_timestamp = 2 [(gogoproto.nullable) = false];
```

Comment:
Can we add a comment indicating at least that this will be ignored for ConditionalPutRequests that contain their own OriginTimestamp.

Reply:
Done. Note that i throw an error, not ignore.